public/assets/js/admin/forms/project_create.js

```diff
@@ -3,6 +3,14 @@ orgCurrencyField = crud.field('org_currency')
 startDateField = crud.field('start_date')
 exchangeRateField = crud.field('exchange_rate')
 
+crud.field('currency').onChange(function(field) {
+    crud.field('get_exchange_rate_button').hide(field.value.toUpperCase() == orgCurrencyField.value).disable(field.value.toUpperCase() == orgCurrencyField.value);
+  }).change();
+
+crud.field('currency').onChange(function(field) {
+    crud.field('exchange_rate').hide(field.value.toUpperCase() == orgCurrencyField.value).disable(field.value.toUpperCase() == orgCurrencyField.value);
+  }).change();
```

Comment:
No need for 2 separate onChange() event handlers here - you can do any number of things inside the function.